### PR TITLE
Misc and stuffs

### PR DESCRIPTION
**Pull request by milipili**

### CODE REVIEW

File: PowerEditor/src/winmain.cpp [hunk 1]
```diff
@@ -37,6 +37,42 @@ namespace
 {
 
 
+void allowVmCopydataMessages(Notepad_plus_Window& notepad_plus_plus, const NppParameters* pNppParameters, winVer ver)
```
Comment (donho) [inline PowerEditor/src/winmain.cpp]: Rather allow**W**mCopydataMessages ?

File: PowerEditor/src/MISC/Process/ProcessAvecThread/Process.h [hunk 2]
```diff
@@ -29,6 +29,7 @@
 #pragma once
 
 #include <windows.h>
+#include "Common.h"
```
Comment (donho) [inline PowerEditor/src/MISC/Process/ProcessAvecThread/Process.h]: Why not? But why?

Comment (donho) [inline PowerEditor/src/MISC/Process/ProcessAvecThread/Process.h]: Why not? But why?

File: PowerEditor/src/winmain.cpp [hunk 3]
```diff
@@ -161,7 +161,7 @@ void parseCommandLine(TCHAR * commandLine, ParamVector & paramVector) {
 	//the commandline generic_string is now a list of zero terminated strings concatenated, and the vector contains all the substrings
 }
 
-bool isInList(const TCHAR *token2Find, ParamVector & params)
+bool removeIfExists(const TCHAR *token2Find, ParamVector & params)
```
Comment (donho) [inline PowerEditor/src/winmain.cpp]: The removal feature is rather for making next parsing easier.
I would say "isInList" is better sementically.

Comment (milipili) [inline PowerEditor/src/winmain.cpp]: `isInList` does not do what it says. You expect something immutable, not something that will change the content of the input. In that regard, this function is badly named.

File: PowerEditor/src/winmain.cpp (hunk 1 above)
Comment (milipili) [inline PowerEditor/src/winmain.cpp]: Oh indeed.

File: PowerEditor/src/MISC/Process/ProcessAvecThread/Process.h (hunk 2 above)
Comment (milipili) [inline PowerEditor/src/MISC/Process/ProcessAvecThread/Process.h]: Please see the commit message:
```
process: add missing include Common.h (generic_string)
```
This include is required for the typedef `generic_string`.

**Comment by milipili** [comment]
It seems that I can't mark the requested changes as done. So doing it here (and rebased from master)

**Comment by donho** [comment]
@milipili Thank you. 
Have you run it with your modification?

**Pull request closed without merging**